R/extract.R

```diff
@@ -9050,3 +9050,96 @@ extract.hurdle <- extract.zeroinfl
 #' @export
 setMethod("extract", signature = className("hurdle", "pscl"),
           definition = extract.hurdle)
+
+# -- extract.logitr (logitr) ---------------------------------------------------
+
+#' @noRd
+extract.logitr <- function(
+  model,
+  include.nobs   = TRUE,
+  include.loglik = TRUE,
+  include.aic    = TRUE,
+  include.bic    = TRUE,
+  ...
+) {
+  summary <- summary(model)
```

Comment:
Perhaps change to `s <- summary(model, ...)`?
Naming objects in the same way as function names (`summary`) is considered bad practice because it might lead to debugging issues.
Added the ellipsis will allow you to change the definition of the `summary` function in your package later and make the functionality available directly here in `texreg` as well.